apps/aecore/lib/aecore/naming/tx/name_revoke_tx.ex

```diff
@@ -41,8 +42,27 @@ defmodule Aecore.Naming.Tx.NameRevokeTx do
   # Callbacks
 
   @spec init(payload()) :: t()
+  # TODO integrate Identifiers
   def init(%{hash: hash}) do
-    %NameRevokeTx{hash: hash}
+    name_hash =
```

Comment:
why does the following have to be so complicated?

Reply:
I will make some tests, it seems like a part of my old ideas, which is not needed anymore.